### PR TITLE
feat: introduce approval reason

### PR DESCRIPTION
**Pull request by rahmatrhd**

### CODE REVIEW

File: approval/repository_test.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (s *RepositoryTestSuite) TearDownTest() {
 }
 
 func (s *RepositoryTestSuite) TestBulkInsert() {
-	expectedQuery := regexp.QuoteMeta(`INSERT INTO "approvals" ("name","index","appeal_id","status","actor","policy_id","policy_version","created_at","updated_at","deleted_at") VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10),($11,$12,$13,$14,$15,$16,$17,$18,$19,$20) RETURNING "id"`)
+	expectedQuery := regexp.QuoteMeta(`NSERT INTO "approvals" ("name","index","appeal_id","status","actor","reason","policy_id","policy_version","created_at","updated_at","deleted_at") VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11),($12,$13,$14,$15,$16,$17,$18,$19,$20,$21,$22) RETURNING "id"`)
```
Comment (StewartJingga) [inline approval/repository_test.go:60]: typo on `NSERT`, should be `INSERT` right?

Comment (StewartJingga) [reply approval/repository_test.go:60]: also why it did not break the test? 🤔

**Review by StewartJingga — CHANGES_REQUESTED** [review]
Found a typo and some bugs probably in the test because the typo does not break the test.

File: approval/repository_test.go (hunk 1 above)
Comment (rahmatrhd) [reply approval/repository_test.go:60]: Maybe because it is regex, will check

Comment (rahmatrhd) [reply approval/repository_test.go]: fixed, `NSERT` was working because it is a regex checking and it still fulfills

**Review by StewartJingga — APPROVED** [review]
LGTM

**Pull request merged**